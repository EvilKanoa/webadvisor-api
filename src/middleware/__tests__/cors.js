const middleware = require('../cors');

jest.mock('cors', () => opts => opts);

describe('cors middleware', () => {
  it('uses the cors middleware with the correct options', () => {
    expect(JSON.stringify(middleware)).toMatchSnapshot();
  });
});
