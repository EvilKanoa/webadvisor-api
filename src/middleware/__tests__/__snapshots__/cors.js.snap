// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cors middleware uses the cors middleware with the correct options 1`] = `"{\\"origin\\":true,\\"credentials\\":true}"`;
